plaidbench/plaidbench/frontend_keras.py

```diff
@@ -242,12 +242,9 @@ def run(self, once=False, warmup=False):
         else:
             epoch_size = self.params.epoch_size
         out = self.model.predict(x=self.x[:epoch_size], batch_size=self.params.batch_size)
-        # Hack to ensure eventlog gets written
-        if not once and not warmup:
-            import keras.backend as b
-            if b.backend() == 'plaidml':
-                b._ctx.shutdown()
-        return (out, {})
+
+        import plaidml.bridge.keras as keras_bridge
+        return (out, {"time": keras_bridge.lastExecuteTime / 1000})
```

Comment:
We need to check if the backend is plaidml before assuming that we can import it. There are cases where we want to run against a different backend and plaidml may not be there.

Reply:
Also, this seems wrong, wouldn't it be `keras_bridge.lastExecuteTimeInMS`?

Reply:
Fixed the name bug.  lastExecuteTime -> lastExecuteTimeInMS.

Reply:
Done.